tests/test_round_trip.rs

```diff
@@ -111,7 +111,7 @@ fn test_round_trip() {
         file.read_to_string(&mut content).unwrap();
 
         let start = Instant::now();
-        let (krate, elapsed) = match content.parse::<syn::Crate>() {
+        let (krate, elapsed) = match syn::parse_str::<syn::File>(&content) {
```

Comment:
I already mentioned this, but this method probably needs to use `parse_file` as well.